chrome-launcher/test/chrome-launcher-test.ts

```diff
@@ -6,59 +6,76 @@
 'use strict';
 
 import {Launcher} from '../chrome-launcher';
-
+import {spy, stub} from 'sinon';
 import * as assert from 'assert';
 
 const log = require('lighthouse-logger');
 const fsMock = {
   openSync: () => {},
-  closeSync: () => {}
+  closeSync: () => {},
+  writeFileSync: () => {}
 };
 
 describe('Launcher', () => {
-  it('accepts and uses a custom path', async () => {
+  beforeEach(() => {
     log.setLevel('error');
```

Comment:
oh i commented on the old closed PR.

basically since log is stateful, if anyone were to set log (which places in the code do sometimes) this essentially resets the log level for each test run.